pyanaconda/modules/subscription/runtime.py

```diff
@@ -265,13 +269,15 @@ def run(self):
             # an organization id being specified RHSM will not know what organization to register
             # the machine. In this throw raise a specific exception so that the GUI can react
             # accordingly and help the user fix the issue.
-
             org_data_task = RetrieveOrganizationsTask(
```

Comment:
I haven't realized you always use the same task class to retrieve the organizations. It would be better to keep the cached list as a class attribute of `RetrieveOrganizationsTask`, so you don't have to propagate it back and forth. The task could have the `reset_cache=False` argument that you could set here to True (as a replacement of `cached_org_data_list=None`).

Reply:
Yeah, that should work and will reduce the amount of boilerplate - just wondering, do we already use something like that for other tasks ? I'm thinking it could be a surprise where the data is coming from later on if this is a one-off.